core/src/main/scala/org/http4s/util/chunk.scala

```diff
@@ -6,13 +6,13 @@ import java.nio.ByteBuffer
 
 trait ChunkInstances {
 
-  implicit def http4sMonoidForChunk[A]: Monoid[Chunk[A]] =
-    new Monoid[Chunk[A]] {
-      // This smells
-      override def combine(x: Chunk[A], y: Chunk[A]): Chunk[A] =
-        (x.toSegment ++ y.toSegment).force.toChunk
+  //Maybe this belongs in fs2?
+  implicit def http4sMonoidForChunk[A]: Monoid[Segment[A, Unit]] =
```

Comment:
The name is now wrong.

If this is still used, it belongs in fs2.  It's not our types.

Reply:
I think I can PR this onto fs2.

Reply:
Also yeah, it's still used, we use `runFoldMonoid` a lot